webapp/src/ts/services/external-app-launcher.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable } from '@angular/core';
```

Comment:
I think for webapp context, yes it makes sense. I've done the name refactor except for where we directly connect with CHT-Android, that is: 
CHT-Android --> webapp
`window.CHTCore.AndroidApi.v1.resolveCHTExternalAppResponse`
and webapp ---> CHT-Android
`window.medicmobile_android.launchExternalApp`

Because it's confusing if we are trying to resolve CHT-Android or another Android app. Maybe because of `CHT-Android` name.